tests-beta/android/MASVS-NETWORK/MASTG-TEST-0x20-1.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Usage of Insecure TLS Protocols
+platform: android
+id: MASTG-TEST-0x20-1
+type: [static]
+weakness: MASWE-0050
+---
+
+## Overview
```

Comment:
Maybe we could start saying something like: 

- "Although the Android Network Security Configuration does not provide direct control over specific TLS versions (unlike iOS https://developer.apple.com/documentation/bundleresources/information_property_list/nsexceptionminimumtlsversion), there are other ways that apps can enable insecure TLS versions:"


Also, should we also include the following? (if yes, please rewrite, this is a copy-paste):

- A note like "In Android 10 and higher, TLS 1.3 is enabled by default for all TLS connections and its cipher suites cannot be customized (therefore, [setEnabledCipherSuites()](https://developer.android.com/reference/javax/net/ssl/SSLSocket#setEnabledCipherSuites(java.lang.String%5B%5D)) is out of scope for Android 10 and higher)"
- An app can obtain an SSLContext that has TLS 1.3 disabled by calling [SSLContext.getInstance("TLSv1.2")](https://developer.android.com/reference/javax/net/ssl/SSLContext#getInstance(java.lang.String)) and can also enable or disable protocol versions on a per-connection basis by calling [setEnabledProtocols()](https://developer.android.com/reference/javax/net/ssl/SSLSocket#setEnabledProtocols(java.lang.String%5B%5D)) on an appropriate object.

Source: https://developer.android.com/privacy-and-security/security-ssl#Updates%20to%20SSL

Reply:
I left out the part with the cipher suites, because this is not content of this test case.